ee/clickhouse/queries/trends/breakdown.py

```diff
@@ -258,8 +258,7 @@ def _parse(result: List) -> List:
                     }
                 )
                 parsed_results.append(parsed_result)
-
-            parsed_result.update({"filter": filter.to_dict()})
+                parsed_result.update({"filter": filter.to_dict()})
```

Comment:
what is this line doing? O.o - we don't need `parsed_result` at this point, do we?